src/goto-symex/symex_function.cpp

```diff
@@ -117,7 +117,7 @@ unsigned goto_symext::argument_assignments(
 
       // Assign value to function argument
       // TODO: Should we hide it (true means hidden)?
-      symex_assign(code_assign2tc(lhs, rhs), true);
+      symex_assign(code_assign2tc(lhs, rhs), false);
```

Comment:
Won't this show the argument assignments when printing the cex?

Reply:
It will, it is an opinionated change.

Reply:
The cex validators historically didn't like this kind of assignment so either revert this change or enable it only when generating an HTML report.

Reply:
This will now only be false when generating a html report.